packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -54,11 +55,26 @@ export class ConfigUtil {
   }
 
   public static async getDefaultUsernameOrAlias(): Promise<string | undefined> {
-    const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
-    const defaultUsernameOrAlias = configAggregator.getPropertyValue(
-      OrgConfigProperties.TARGET_ORG
-    );
-    return defaultUsernameOrAlias ? String(defaultUsernameOrAlias) : undefined;
+    try {
+      const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
+      const defaultUsernameOrAlias = configAggregator.getPropertyValue(
+        OrgConfigProperties.TARGET_ORG
+      );
+      if (defaultUsernameOrAlias === undefined) {
+        return undefined;
+      }
+
+      return JSON.stringify(defaultUsernameOrAlias).replace(/\"/g, '');
+    } catch (err) {
+      console.error(err);
+      if (err instanceof Error) {
+        TelemetryService.getInstance().sendException(
+          'get_default_username_alias',
+          err.message
+        );
+      }
+      return undefined;
```

Comment:
Instead of catching the exception and returning undefined, it's probably better to re-throw the error.